public/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+  "short_name": "InBrowser IPFS",
+  "name": "InBrowser IPFS Gateway",
```

Comment:
:thought_balloon: ideas for better name welcome. We need something short, perhaps just go with `inbrowser.link` ?

Reply:
i think thats about as short as we can go and have it still be semantic